azext_iot/tests/iothub/core/test_iothub_storage_int.py

```diff
@@ -376,3 +402,34 @@ def test_user_identity_storage(self):
             )
 
         self.tearDown()
+
+    def wait_till_job_completion(self, job_id):
+        tries = 0
+        status = self.cmd(
```

Comment:
Instead of running the command here, you can simply assign status = "running"

Reply:
I had it originally as status is nothing but then I saw that for export/import jobs, it can be almost instantaneous - as in this call would return a completed.

Reply:
I see your point, then why not do something like:

```
while tries < JOB_POLL_MAX_ATTEMPTS:
        status_output = self.cmd(
            f"iot hub job show -n {self.entity_name} -g {self.entity_rg} --job-id {job_id}"
        ).get_output_in_json()
        status = status_output["status"]
        if status in ["failed", "completed"]:
            break;
        sleep(SETUP_SLEEP_INTERVAL)
        tries += 1
```

Reply:
updated loop